mlprimitives/jsons/sklearn.preprocessing.RobustScaler.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "sklearn.preprocessing.RobustScaler",
+  "contributors": ["Hector Dominguez <hedb_12@hotmail.com>"],
+  "documentation": "http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.RobustScaler.html",
+  "description": "Scale features using statistics that are robust to outliers",
+  "classifiers": {
+    "type": "preprocessor",
+    "subtype": "transformer"
+  },
+  "modalities": [],
+  "primitive": "sklearn.preprocessing.RobustScaler",
+  "fit": {
+    "method": "fit",
+    "args": [
+      {
+        "name": "X",
+        "type": "ndarray",
+        "description": "The data used to compute the median and quantiles used for later scaling along the features axis"
+      }
+    ]
+  },
+  "produce": {
+    "method": "transform",
+    "args": [
+      {
+        "name": "X",
+        "type": "ndarray"
+      }
+    ],
+    "output": [
+      {
+        "name": "X",
+        "type": "ndarray"
+      }
+    ]
+  },
+  "hyperparameters": {
+    "fixed": {
+      "with_centering": {
```

Comment:
This should be a tunable hyperparameter.
Depending on the type of data and estimator, centering will be more or less effective in terms of  estimation performance.